harness_cloud_cost_management/manifest.json

```
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3eb2e9ef-2c9c-45b6-8f1c-8a900910f948",
+  "app_id": "harness-cloud-cost-management",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Cloud Cost Management allows Harness customers to access and view their cloud and cluster cost metrics over a specified range of dates",
+    "title": "Harness Cloud Cost Management",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Cloud Cost Management Dashboard on DataDog",
+        "image_url": "images/ccm_dashboard_on_datadog.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Cloud Cost Management Dashboard on Harness",
+        "image_url": "images/ccm_dashboard_on_harness.png"
+      }
+    ]
+  },
+  "author": {
+    "homepage": "https://www.harness.io",
+    "support_email": "akash.bhardwaj@harness.io",
+    "vendor_id": "TODO Please Fill In"
+  },
+  "oauth": {},
+  "assets": {
+    "dashboards": {
+      "harness_cloud_cost_management_overview": "assets/dashboards/harness_cloud_cost_management_overview.json"
+    },
+    "integration": {
+      "source_type_name": "Harness Cloud Cost Management",
+      "configuration": {},
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "harness_cloud_cost_management.",
+        "check": "",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
+    },
+    "logs": {}
+  },
+  "legal_terms": {}
```

```suggestion
    }
```

Can delete the whole integration section.